src/System.Globalization.Calendars/System.Globalization.Calendars.Tests.sln

```diff
@@ -0,0 +1,22 @@
+﻿
```

Comment:
The .sln should be System.Globalization.Calendars.sln, not System.Globalization.Calendars.Tests.sln.